plugins/transport-nio/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -21,3 +21,18 @@ grant codeBase "${codebase.elasticsearch-nio}" {
   // elasticsearch-nio makes and accepts socket connections
   permission java.net.SocketPermission "*", "accept,connect";
 };
+
+grant codeBase "${codebase.netty-common}" {
+   // for reading the system-wide configuration for the backlog of established sockets
+   permission java.io.FilePermission "/proc/sys/net/core/somaxconn", "read";
+
+   // This should only currently be required as we use the netty http client
+   // netty makes and accepts socket connections
+   permission java.net.SocketPermission "*", "accept,connect";
+};
+
+grant codeBase "${codebase.netty-transport}" {
+   // Netty NioEventLoop wants to change this, because of https://bugs.openjdk.java.net/browse/JDK-6427854
+   // the bug says it only happened rarely, and that its fixed, but apparently it still happens rarely!
+   permission java.util.PropertyPermission "sun.nio.ch.bugLevel", "write";
```

Comment:
LOL oh boy. I thought we don't need it and it will just log a warning from the netty end?